robots/cmd/kubevirtci-bumper/main.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2010 The KubeVirt Authors.
+Copyright 2020 The KubeVirt Authors.
```

Comment:
2021? :)

Reply:
The file was originally created in 2020: https://github.com/kubevirt/project-infra/blame/79deeabbfe1d1776a2549cc7482beee7ceab23e9/robots/cmd/kubevirtci-bumper/main.go